src/IRrecv.cpp

```diff
@@ -1166,7 +1166,7 @@ bool IRrecv::matchAtLeast(uint32_t measured, uint32_t desired,
   DPRINT(". Matching: ");
   DPRINT(measured);
   DPRINT(" >= ");
-  DPRINT(ticksLow(std::min(desired, MS_TO_USEC(params.timeout)), tolerance,
+  DPRINT(ticksLow(std::min(desired, (uint32_t)MS_TO_USEC(params.timeout)), tolerance,
```

Comment:
src/IRrecv.cpp:1169:  Lines should be <= 80 characters long  [whitespace/line_length] [2]

Reply:
done :-)